mesh/api/api_test.go

```diff
@@ -253,11 +279,13 @@ func TestMeshGraph(t *testing.T) {
 		t.Fatal(err)
 	}
 	actual, _ := io.ReadAll(resp.Body)
+	t.Logf("Actual response body: %s", string(actual)) // Print the actual variable
+
 	expected, _ := os.ReadFile("testdata/test_mesh_graph.expected")
-	if runtime.GOOS == "windows" {
-		expected = bytes.Replace(expected, []byte("\r\n"), []byte("\n"), -1)
-	}
-	expected = expected[:len(expected)-1] // remove EOF byte
+	//if runtime.GOOS == "windows" {
+	//	expected = bytes.Replace(expected, []byte("\r\n"), []byte("\n"), -1)
+	//}
+	//expected = expected[:len(expected)-1] // remove EOF byte
```

Comment:
Does this need to be removed?

Reply:
Probably not, it was something I added at the beginning, then I found why it was failing, forgot to remove it. Thanks.